spaceship/lib/spaceship/tunes/availability.rb

```diff
@@ -111,6 +107,15 @@ def add_b2b_users(user_list = [])
         end
         self
       end
+
+      # Updates users for b2b enabled apps
+      def update_b2b_users(users_to_add: [], users_to_remove: [])
+        raise "Cannot add b2b users if b2b is not enabled" unless b2b_app_enabled
+        @b2b_users = b2b_users.reject { |user| users_to_remove.include?(user.ds_username) }
+        @b2b_users.concat(users_to_add.map { |user| B2bUser.from_username(user) })
```

Comment:
Do we need to make sure the user isn't already added? Or will we let that responsibility fall onto the user?

Reply:
I ll try to incorporate that check.